examples_test.go

```diff
@@ -31,15 +30,16 @@ func Example_timeout() {
 	// Do not set a timeout on the context passed to NewClient: dialing happens
 	// asynchronously, and the context is used to refresh credentials in the
 	// background.
-	client, err := bigquery.NewClient(ctx, "project-id")
+	client, err := secretmanager.NewClient(ctx)
 	if err != nil {
 		// TODO: handle error.
 	}
 	// Time out if it takes more than 10 seconds to create a dataset.
 	tctx, cancel := context.WithTimeout(ctx, 10*time.Second)
 	defer cancel() // Always call cancel.
 
-	if err := client.Dataset("new-dataset").Create(tctx, nil); err != nil {
+	req := &secretmanagerpb.DeleteSecretRequest{Name: "projects/proj/secrets/name"}
```

Comment:
Make this more clear it should be your project ID? There might be a better way.

```suggestion
	req := &secretmanagerpb.DeleteSecretRequest{Name: "projects/project-id/secrets/name"}
```

Reply:
Good idea.